### DISCUSSION TITLE
Is it necessary to add a parameter to ignore the convergence check?

### DISCUSSION THREAD

**Discussion by panxiang126** [deepmodeling/dpdata#350] [Ideas]
When I extracted the results of vasp using dpdata, it automatically ignored some unconverged data, which was reasonable for the next training with deepmd. But if I only want to extract the trajectories, some structures will be missed. Therefore, I am wondering if it is necessary to add an option to ignore the convergence check and provide a warning.

**Comment by wanghan-iapcm** [comment]
Good suggestion, could you please commit a PR for it?

**Reply by panxiang126** [reply]
Okay.
